source/code/plugins/filter_syslog.rb

```diff
@@ -30,6 +30,7 @@ def filter(tag, time, record)
       # Use Time.now, because it is the only way to get subsecond precision in version 0.12.
       # The time may be slightly in the future from the ingestion time.
       record["Timestamp"] = OMS::Common.format_time(Time.now.to_f)
+      record["EventTime"] = OMS::Common.format_time(time)
```

Comment:
I checked on the machine you shared with me. The /var/log/syslog file was actually logging UTC timestamps, whereas Time.strptime in ruby - which is what fluentd uses to parse the string assumes local time. So, for example, the syslog file says 19:05 - which is the actual UTC time but strptime assumes it is Asia/Jerusalem time (/etc/timezone) and makes the UTC time as 16:05. I restarted the syslog daemon (sudo service rsyslogd restart) and now it is logging local time and fluentd's timestamp is also fixed. 

So, I guess if you change the timezone on the machine you need to restart the syslog daemon